var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -113,7 +116,7 @@ def url_for_version(self, version):
     depends_on('py-numpy', when='+python')
     depends_on('swig', when='+python')
 
-    patch('umfpack_from_suitesparse.patch')
+    patch('umfpack_from_suitesparse.patch', when='@:12.8.1')
```

Comment:
Actually it is not needed for the master/develop version of Trilinos, how would I indicate that?